src/components/navBar/navBar.css

```diff
@@ -0,0 +1,96 @@
+.navBar-container {
+    display: flex;
+    flex-direction: row;
+    width: 1440px;
+    height: 53px;
+    padding: 14px 123px 5px;
+    background-color: var(--white);
+}
+
+.logo-icon {
+    width: 33px;
+    height: 33px;
+    margin-left: 123px;
+    margin: 0 15px 1px 0;
+    object-fit: contain;
+}
+
+.logo{
+    width: 115px;
+    height: 32px;
+    margin: 1px 31px 1px 15px;
+    font-family: HelveticaNeue;
+    font-size: 27px;
```

Comment:
Avoid using pixels when referencing anything that has to do with fonts and text. Use rem instead.
**Reference** - https://medium.com/@julienetienne/pixels-are-dead-faa87cd8c8b9